snapcraft/internal/lifecycle/_runner.py

```diff
@@ -59,6 +59,13 @@ def execute(step, project_options, part_names=None):
     :returns: A dict with the snap name, version, type and architectures.
     """
     config = project_loader.load_config(project_options)
+    if project_options.info.vendoring:
+        logger.warning(
+            'Vendoring is set to {} but it will not be enforced. '
+            'Snapcraft must build inside a LXD container to setup a proxy '
+            'that limits access to external resources to the specified '
+            'hosts. The build will proceed without restrictions.'.format(
```

Comment:
This is better, but there's too much technical detail, and the actual hostnames don't matter, either. How about "This snap uses vendoring, but vendoring cannot be enforced without using cleanbuild. Proceeding without restriction."